tezos/api/Cargo.toml

```diff
@@ -10,7 +10,7 @@ enum-iterator = "0.6"
 failure = "0.1"
 chrono = { version = "0.4", features = ["serde"] }
 lazy_static = "1.4"
-ocaml = "0.9.3"
+znfe = { git = "https://github.com/tizoc/znfe.git" }
```

Comment:
@tizoc 
I started discussion on slack about this a how to version it :)

Reply:
Ok, now it points to a specific tag.